extensions/adobe/experience/campaign-experienceevent-v2.schema.json

```diff
@@ -0,0 +1,172 @@
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField5": {
+              "title": "Custom Field 5",
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField6": {
+              "title": "Custom Field 6",
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField7": {
+              "title": "Custom Field 7",
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField8": {
+              "title": "Custom Field 8",
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField9": {
+              "title": "Custom Field 9",
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField10": {
+              "title": "Custom Field 10",
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField11": {
+              "title": "Custom Field 11",
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField12": {
+              "title": "Custom Field 12",
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField13": {
+              "title": "Custom Field 13",
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField14": {
+              "title": "Custom Field 14",
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField15": {
+              "title": "Custom Field 15",
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField16": {
+              "title": "Custom Field 16",
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField17": {
+              "title": "Custom Field 17",
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField18": {
+              "title": "Custom Field 18",
+              "type": "string"
+            },
+            "xdm:cusField19": {
+              "title": "Custom Field 19",
+              "description": "",
+              "type": "string"
+            },
+            "xdm:cusField20": {
+              "title": "Custom Field 20",
+              "description": "",
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/extensible#/definitions/@context"
+    },
+    {
+      "ref": "https://ns.adobe.com/xdm/experience/campaign/experienceevent-all-v2"
+    },
+    {
+      "$ref": "https://ns.adobe.com/experience/decisioning/proposition"
+    },
+    {
+      "$ref": "#/definitions/campaign-experienceevent-v2"
```

Comment:
The local definition from above is named "campaign-experienceevent". This is what you need to reference here.

Reply:
fixed